t.cs

```diff
@@ -0,0 +1,40 @@
+public class Test
+{
+    public bool Enabled
+    {
+        get
+        {
+
+            return true;
+        }
+    }
+
+    public bool T(bool c)
+    {
+        do
+        {
+            Console.WriteLine("Loop forever");
+        }
+
+        while (true);
+
+        if (c)
+        {
+            T(false);
+        }
+        return true;
+        // Return the value of the 'enabled' field.
```

Comment:
Single-line comment should be preceded by blank line.

```suggestion

        // Return the value of the 'enabled' field.
```